mitodl/odl-video-service
# Make TechTV URLs work with or without slugs

#### What are the relevant tickets?
Closes #494 

#### What's this PR do?
- Makes a TechTV video URL work with or without a title slug

#### How should this be manually tested?
- Import TechTV collection #3033 first ([instructions here](https://github.com/mitodl/odl-video-service/pull/429))
- The following URL's should all work:
  - `/videos/3033`
  - `/videos/3033-something`
  - `/embeds/3033`
  - `/embeds/3033-something`
